test/integration_test/specs/mysql-backup/backuplocation.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
```

Comment:
This can't be here, the test needs to read this from some configmap and create the backuplocation.

Reply:
Fixed.